tests/unit/components/pacbio/PacbioLibraryCreate.spec.js

```diff
@@ -45,20 +45,21 @@ describe('PacbioLibraryCreate.vue', () => {
   })
 
   it('must have tagOptions data', () => {
-    expect(modal.tagOptions).toEqual([])
+    expect(modal.tagOptions).toEqual([{ value: '', text: 'No tag' }])
   })
 
   describe('#createLibrary', () => {
-    let payload
+    let payload, payload_no_tag
```

Comment:
As far as I can see payload_no_tag is only used in 1 test so you could declare it in the test. Also the preferred case type in JavaScript is camelCase.